weather/app/views/cityWeather.scala.html

```diff
@@ -1,29 +1,35 @@
+@(weather: models.accuweather.WeatherResponse)(implicit request: RequestHeader)
+
+@import common.Edition
+
 <div class="weather">
     <div class="u-unstyled weather__current js-weather-current">
         <p class="weather__desc">
             <span class="u-h">The temperature</span>
             <span class="weather__time">Now</span>
             <span class="u-h">is</span>
-            <span class="weather__temp js-weather-temp">{{tempNow}}</span>
+            @defining(weather.temperatureForEdition(Edition(request))) { temp =>
+                <span class="weather__temp js-weather-temp">@math.round(temp.Value)°@temp.Unit</span>
+            }
         </p>
-        <i class="i i-weather-{{icon}} weather__icon js-weather-icon" title="{{description}}"><span class="u-h">{{description}}</span></i>
+        @fragments.inlineSvg(s"weather-${weather.WeatherIcon}", "weather", Seq("weather__icon","js-weather-icon"))<span class="u-h">@weather.WeatherText</span>
```

Comment:
I would rather use '.weather__icon', as it fits the structure of the component, and remove the 'inline-weather'.